modules/apps/frontend-js/frontend-js-web/src/main/resources/META-INF/resources/liferay/liferay.d.ts

```diff
@@ -165,7 +165,7 @@ declare module Liferay {
 			data: Object
 		): void;
 
-		export function sub(langKey: string, ...args: any[]): string;
+		export function sub(string: string, ...args: any[]): string;
```

Comment:
I think instead of `any[]`, it can be 

```typescript
type TSubArg = string | function
export function sub(string: string, ...args: TSubArg[]): string;
```

Reply:
> `type TSubArg = string | function`

Why do you think it can be a string or function? We are checking against an object and function in the util. If anything, replacing `any[]` could be `string[]`.

Reply:
> Why do you think it can be a string or function?

Nice question. I don't remember, I spent 20min thinking but I don't remember. If I remember, I'll share with you this delirium 😂 😂 

you can use `string[]` instead of `any[]`